.ci/test.sh

```diff
@@ -74,10 +74,9 @@ if [[ $TASK == "lint" ]]; then
         ${CONDA_PYTHON_REQUIREMENT} \
         cmakelint \
         cpplint \
-        isort \
```

Comment:
`pre-commit` installs packages into its own virtualenv, so it's not necessary to pre-install them any more